### PR TITLE
Waiv 1581,1582

### PR DESCRIPTION
**Pull request by oksanana**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

**Comment by oksanana** [comment]
ready only 1582, still work with 1581

File: src/client/components/Story/Body.js [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 import steemEmbed from '../../vendor/embedMedia';
 import { jsonParse } from '../../helpers/formatter';
 import sanitizeConfig from '../../vendor/SanitizeConfig';
 import { imageRegex, dtubeImageRegex, rewriteRegex } from '../../helpers/regexHelpers';
 import htmlReady from '../../vendor/steemitHtmlReady';
 import improve from '../../helpers/improve';
 import PostFeedEmbed from './PostFeedEmbed';
+import DetailsBody from '../../rewards/Details/DetailsBody';
 import './Body.less';
+import { getFieldWithMaxWeight } from '../../object/wObjectHelper';
```
Comment (milklim) [inline src/client/components/Story/Body.js:18]: reorder imports

File: src/client/components/Story/Body.js [hunk 2]
```diff
@@ -97,29 +100,66 @@ export function getHtml(body, jsonMetadata = {}, returnType = 'Object', options
   return <div dangerouslySetInnerHTML={{ __html: sections.join('') }} />;
 }
 
+const getObjectDetails = metaData => {
+  try {
+    if (!metaData) return {};
+    const parsedJsonMetadata = jsonParse(metaData) || {};
+    return get(parsedJsonMetadata, 'waivioRewards.proposition') || {};
```
Comment (milklim) [inline src/client/components/Story/Body.js:107]: use array for prop path: get(parsedJsonMetadata, ['waivioRewards', 'proposition'])

File: src/client/components/Story/Body.js [hunk 3]
```diff
@@ -97,29 +100,66 @@ export function getHtml(body, jsonMetadata = {}, returnType = 'Object', options
...
+  } catch (e) {
+    return {};
+  }
+};
+
+const getProposedWobj = metaData => {
+  try {
+    if (!metaData) return {};
+    const parsedJsonMetadata = jsonParse(metaData) || {};
+    return get(parsedJsonMetadata, 'waivioRewards.proposedWobj') || {};
```
Comment (milklim) [inline src/client/components/Story/Body.js:117]: same

**Pull request merged**